PluginsAndFeatures/azure-toolkit-for-intellij/resources/whatsnew/whatsnew.md

```diff
@@ -1,60 +1,107 @@
-
-3. Select the right subscription and cluster you want, and then select *Create app...* and input an unique name like `gs-spring-boot-azure` for the app.
-
-
-![image](https://user-images.githubusercontent.com/17075863/77133144-f3720600-6a9c-11ea-898a-f4c8cf3c0183.png)
-
-
-4. Start the deploy by clicking *Run* Button. The plugin will run *mvn package* on the project first and then create the new app and deploy the jar generated by *package* step.
-
-![image](https://user-images.githubusercontent.com/17075863/77133334-b35f5300-6a9d-11ea-9552-bfaae43193cd.png)
-
-
-#####  Open Azure Spring Cloud app in Azure Explorer
-1. Click the Azure Explorer button at left side bar and sign-in to Azure
-
-![image](https://user-images.githubusercontent.com/17075863/77133434-06d1a100-6a9e-11ea-91f9-123efb4258d9.png)
-
-2. Find the *Spring Cloud* node and double click.
-
-![image](https://user-images.githubusercontent.com/17075863/77133505-3bddf380-6a9e-11ea-924e-221e02ee1ac6.png)
-
-3. Wait a few seconds and you will see all Azure Spring Cloud Clusters in the tree, double click the cluster you just created app to, wait a few seconds for the loading, you will be able to see the new app you just created.
-
-4. Right click the app you have just created. Select *Show Properties*, you will see the property view for app.
-
-![image](https://user-images.githubusercontent.com/17075863/77133618-a5f69880-6a9e-11ea-9349-9b53c3c809e8.png)
-
-##### Navigate in browser
-1. In the property view, you will see 'N/A' in the URL, this is because by default the *public endpoint* is not enabled, you can assign endpoint which can be accessed publicly by clicking `Assign endpoint` button.
-
-![image](https://user-images.githubusercontent.com/17075863/77134962-62069200-6aa4-11ea-9cbc-4ea7fe218ac0.png)
-
-2. Wait a few seconds for the `Assign endpoint` operation, you will see the URL field is showing a clickable link. Click this link will open a browser and show you the message *Greetings from Spring Boot*.
-
-![image](https://user-images.githubusercontent.com/17075863/77135088-e0fbca80-6aa4-11ea-8255-9239a5e85e3b.png)
-
-#### Add Azure specific dependencies
-You need to read this [tutorial](https://docs.microsoft.com/en-us/azure/spring-cloud/spring-cloud-tutorial-prepare-app-deployment) for Azure specific dependencies, to simplify the steps to configure dependencies, this plugin provides the dependency management for you. Open the *pom.xml* file in Intellij, right click on the editor, select *Add Azure Spring Cloud dependency* menu, wait a few minutes for the dependency resolving, it will update your *pom.xml* file with the compatible latest version mentioned in this [tutorial](https://docs.microsoft.com/en-us/azure/spring-cloud/spring-cloud-tutorial-prepare-app-deployment) 
-
-![image](https://user-images.githubusercontent.com/17075863/77284376-7d260b80-6d09-11ea-8cc0-769a3fdc169f.png)
-
+<!-- Version: 3.37.0 -->
+# What's new in Azure Toolkit for IntelliJ
+
```

Comment:
Find the old features hard to be notices with 3.37 on top, so let's add one sentence in the very beginning for this release:

Scroll down to checkout our newly added support for [Azure Functions](to section below), [Azure Spring Cloud](to section below) and [log streaming for Azure App Service](to section below).

Reply:
Like add a toc at the top?

``` md
- [New Features]()
  - [App Service Log Streaming](#3360)
  - [Azure Spring Cloud Support](#3350)
  - [Azure Function Support](#3340)
- [Change Log](#change-log)
  - [3.37.0](#3370current-release)
  - [3.36.0](#3360)
  - [3.35.0](#3350)
  - [3.34.0](#3340)
```